lib/src/package_config.dart

```diff
@@ -227,11 +227,11 @@ abstract class Package {
   /// If [extraData] is supplied, it will be available as the
   /// [Package.extraData] of the created package.
   factory Package(String name, Uri root,
-          {Uri /*?*/ packageUriRoot,
-          LanguageVersion /*?*/ languageVersion,
+          {Uri? packageUriRoot,
+          LanguageVersion? languageVersion,
           dynamic extraData}) =>
```

Comment:
```suggestion
          Object? extraData}) =>
```

(throughout if you accept the suggestion)

Reply:
LGTM.